### ISSUE TITLE
nothing in .fa file when converting fastq to fasta

### ISSUE THREAD

**Issue by climbandseq** [lh3/seqtk#113]
Hello,
I am very (!) new to working with sequencing files as well as Linux command line.

I used the command seqtk seq -a test.fq.gz > out.fa to try to convert a fastq file to fasta. I created the out.fa file but there is nothing in it. I also tried seqtk seq -A test.fq.gz > out.fa but with the same result. 

I am sure it is something simple. I get no errors, and a .fa file is created, but there is nothing in the file.
Thanks for your help!

**Comment by tseemann** [comment]
The command `seqtk seq -A file.fq.gz > out.fa` looks correct.

Maybe you `file.fq.gz` is empty?
What does `gzip -c -d file.fq.gz | head -n 4` say?

What version are you using?
```
seqtk 2>&1 | grep Version
Version: 1.2-r94
```

**Comment by evolgenomology** [comment]
Try committing the .gz ending.

On 3 May 2018 at 15:52:51, climbandseq (notifications@github.com) wrote:

.gz

**Comment by climbandseq** [comment]
Thanks guys! I think I figured it out, though. 
When I created the test.fq.gz file, I copied one of my fastq files out of another folder and renamed it to make it simple. This morning I tried the following commands to see if maybe renaming the file was a problem. I did the following:
cd /mnt/c/Users/"sequencer PC"/Documents/Location/Run1/Fastq
seqtk seq -A 2_S1_L001_R1_001.fastq.gz > out.fa
And it worked! I got my fasta file with content. Not sure if renaming the file had something to do with that.

Is there a single command to convert every fastq file in that folder to fasta? For example, can I say:
seqtk seq -A *.fastq.gz > out.fa  
Or something similar to convert all files in that folder that end in fastq.gz to fasta.

Thanks for your help! Not sure if renaming the file caused the problem.
